### PR TITLE
feat: improve charts and graphs

### PR DESCRIPTION
**Pull request by eloy-encord**

### CODE REVIEW

File: encord_consensus/lib/generate_charts.py [hunk 1]
```diff
@@ -78,3 +78,57 @@ def get_line_chart(data: dict, title: str, x_title: str, y_title: str) -> alt_ap
         labelFontSize=20,
         titleFontSize=20,
     )
+
+
+def get_label_occurrence_per_frame_chart(
```
Comment (alex-encord) [inline encord_consensus/lib/generate_charts.py:84]: Is there a way we can get rid of the
```UserWarning: I don't know how to infer vegalite type from 'empty'.  Defaulting to nominal.```
warning?

Comment (eloy-encord) [reply encord_consensus/lib/generate_charts.py]: Thanks for noticing, found the offending cause, one of the merged charts could be empty at merge time. Could you mind check again?

Comment (alex-encord) [reply encord_consensus/lib/generate_charts.py]: Works without warnings! 🌟

**Review by alex-encord — APPROVED** [review]
Nice! 👨‍🎨

**Pull request merged**